src/components/ExportScriptFlyout/Body.tsx

```diff
@@ -28,18 +28,18 @@ import type { Setter } from '../../common/types';
 
 interface Props {
   code: string;
-  exportAsSuite: boolean;
-  setExportAsSuite: Setter<boolean>;
+  exportAsProject: boolean;
+  setExportAsProject: Setter<boolean>;
 }
 
-export function Body({ code, exportAsSuite, setExportAsSuite }: Props) {
+export function Body({ code, exportAsProject, setExportAsProject }: Props) {
   return (
     <EuiFlyoutBody>
       <EuiCheckbox
-        id="export-as-suite-checkbox"
-        label="Export as suite"
-        checked={exportAsSuite}
-        onChange={() => setExportAsSuite(!exportAsSuite)}
+        id="export-as-project-checkbox"
```

Comment:
TBH I'm unsure why this property is even here. Perhaps it's required by EUI. In any case, it's fine to rename as it's unreferenced elsewhere in the code.

Reply:
Yes, id is required for EuiCheckbox component